test/foundry/unit/fuzz/TestFuzz_ERC4337Account.t.sol

```diff
@@ -46,13 +46,13 @@ contract TestFuzz_ERC4337Account is NexusTest_Base {
         vm.assume(numOps < 20); // Keep the number of operations manageable
 
         for (uint256 i = 0; i < numOps; i++) {
-            uint256 nonceBefore = getNonce(address(BOB_ACCOUNT), MODE_VALIDATION, address(VALIDATOR_MODULE));
+            uint256 nonceBefore = getNonce(address(BOB_ACCOUNT), MODE_VALIDATION, address(VALIDATOR_MODULE), bytes3(0));
```

Comment:
maybe use some random batchId since it's fuzz

Reply:
ok